spec.html

```diff
@@ -21458,6 +21611,18 @@
   </ul>
   <p>An implementation may define behaviour other than throwing *RangeError* for `toFixed`, `toExponential`, and `toPrecision` when the _fractionDigits_ or _precision_ argument is outside the specified range.</p>
 
+  <emu-clause id="sec-host-report-errors" aoid="HostReportErrors">
+    <h1>HostReportErrors( _errorList_ )</h1>
+
+    <p>HostReportErrors is an implementation-defined abstract operation that allows host environments to report parsing errors, early errors, and runtime errors.</p>
+
+    <p>An implementation of HostReportErrors must complete normally in all cases. The default implementation of HostReportErrors is to do nothing.</p>
+
+    <emu-note>
+      _errorList_ will be a List of JavaScript values. If the errors are parsing errors or early errors, these will always be *SyntaxError* or *ReferenceError* objects. Runtime errors, however, can be any ECMAScript value.
```

Comment:
I think for host-implemented abstract ops it's generally important to have a non-normative note explaining what the inputs are, so that hosts can easily write correct implementations.